common/src/main/java/tachyon/security/LoginUser.java

```diff
@@ -64,6 +64,19 @@ public static User get(TachyonConf conf) throws IOException {
     return sLoginUser;
   }
 
+  public static void setLoginUser(User user) {
+    sLoginUser = user;
+  }
+
+  /**
+   * Create a user for testing Tachyon
+   * @param user the full user principal name
+   * @return a fake user for running unit tests
+   */
+  public static User createUserForTesting(String user) {
```

Comment:
This methods was used to run unit tests with some specific login user. I know Qifan created a new feature TACHYON-786(Add CUSTOM_LOGIN loginmodule). I will use his feature when it has been committed. Thanks for your comments